user/example/misc/example_startrun.sh

```diff
@@ -5,11 +5,11 @@ $BINPATH/euRun -n Ex0RunControl &
 sleep 1
 $BINPATH/euLog &
 sleep 1
-$BINPATH/euCliMonitor -n Ex0Monitor -t my_mon & 
+$BINPATH/euCliMonitor -n Ex0Monitor -t my_mon &
 $BINPATH/euCliCollector -n Ex0TgDataCollector -t my_dc &
 # The following data collectors are provided if you build user/eudet
-#$BINPATH/euCliCollector -n DirectSaveDataCollector -t my_dc &
-#$BINPATH/euCliCollector -n EventIDSyncDataCollector -t my_dc &
-#$BINPATH/euCliCollector -n TriggerIDSyncDataCollector -t my_dc &
+$BINPATH/euCliCollector -n DirectSaveDataCollector -t my_dc &
+$BINPATH/euCliCollector -n EventIDSyncDataCollector -t my_dc &
+$BINPATH/euCliCollector -n TriggerIDSyncDataCollector -t my_dc &
```

Comment:
The -t <> option has to provide a unique name.

Reply:
I don't remember doing this change but maybe I simply misremember? But I now called them something else specific  to the type of data collector.